assets/quick-add.css

```diff
@@ -78,6 +82,10 @@
   max-width: 100%;
```

Comment:
Nitpick but spacing feels a bit wide here 🤔 Want to ensure it's intentional 
https://screenshot.click/03-20-ihglj-stnst.png

Reply:
Yeah it feels like too much to me too, but I picked the value we were using already for Featured Products:
<img width="1056" alt="image" src="https://github.com/Shopify/dawn/assets/590055/9e903158-8113-4d5a-ab97-2eee9cc0b6d3">

So it was intentional, but lemme know if you think we should run it by UX?

Reply:
That is fair 👍 I was just comparing it to what we had before: https://screenshot.click/03-17-lla7m-blvm9.png